usaspending_api/awards/v2/data_layer/orm.py

```diff
@@ -41,6 +41,8 @@ def construct_assistance_response(requested_award_dict):
         return None
     response.update(award)
 
+    response["executive_details"] = create_officers_object(award, FABS_ASSISTANCE_FIELDS, "fabs")
```

Comment:
Wouldn't it make more sent to include in the list of `response["<key>"]` lines? What is the justification for segregating at the top?

Reply:
Consistency with the other responses. It struck me as strange as well, but I assumed it was better to minimize changes once in flight.

Reply:
Sorry, I answered the wrong question before. I seperated this line because it no longer makes use of the transaction object like everything else does (since under current business logic the "latest transaction" can mean something different for EC and other fields), so that future devs don't glance over this line and assume that if just pulls a field from the transaction variable like everything else.